Mathlib/MeasureTheory/Integral/SetIntegral.lean

```diff
@@ -477,39 +477,39 @@ theorem integral_indicatorConstLp [CompleteSpace E]
 set_option linter.uppercaseLean3 false in
 #align measure_theory.integral_indicator_const_Lp MeasureTheory.integral_indicatorConstLp
 
-theorem set_integral_map {β} [MeasurableSpace β] {g : α → β} {f : β → E} {s : Set β}
+theorem set_integral_map {β} [MeasurableSpace β] {g : X → β} {f : β → E} {s : Set β}
     (hs : MeasurableSet s) (hf : AEStronglyMeasurable f (Measure.map g μ)) (hg : AEMeasurable g μ) :
     ∫ y in s, f y ∂Measure.map g μ = ∫ x in g ⁻¹' s, f (g x) ∂μ := by
   rw [Measure.restrict_map_of_aemeasurable hg hs,
     integral_map (hg.mono_measure Measure.restrict_le_self) (hf.mono_measure _)]
   exact Measure.map_mono_of_aemeasurable Measure.restrict_le_self hg
 #align measure_theory.set_integral_map MeasureTheory.set_integral_map
 
-theorem _root_.MeasurableEmbedding.set_integral_map {β} {_ : MeasurableSpace β} {f : α → β}
+theorem _root_.MeasurableEmbedding.set_integral_map {β} {_ : MeasurableSpace β} {f : X → β}
```

Comment:
This lemma now involves two measurable spaces of which one is called `X` and the other `β`.

Reply:
I'm happy to rename \beta to Y also, if that's desired.

Reply:
We definitely want the naming be consistent, so either both names should change or neither.